cpp/examples/basic/Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM nvidia/cuda:11.1.1-devel-ubuntu18.04
+ARG PARALLEL_LEVEL=8
```

Comment:
Should move this down to line 35 to avoid re-running all the `apt install` if someone changes this build arg.

Reply:
Could you clarify, is it just line 2 or both line 1, 2? Can't quite tell from gh's UI.